docker-compose.yml

```diff
@@ -0,0 +1,54 @@
+version: '3'
+
+services:
+  endpoint_querier:
+    container_name: endpoint_querier_1
```

Comment:
why `_1`?

Reply:
I thought this will better prepare us for when we have multiple endpoint queriers